package.json

```diff
@@ -602,6 +653,29 @@
           "type": "boolean",
           "default": false,
           "description": "Specifies whether notifications should be shown if OmniSharp encounters warnings or errors loading a project. Note that these warnings/errors are always emitted to the OmniSharp log"
+        },
+        "omnisharp.preview": {
```

Comment:
This doesn't already exist? @rchande / @akshita31 / @TheRealPiotrP I thought you guys already had an `omnisharp.preview` flag that we were sitting behind?

Reply:
@NTaylorMullen We didn't have a flag like that before.

Reply:
Ah, alright. Well 👍